src/app/services/storage.py

```diff
@@ -26,70 +26,87 @@ class S3Bucket:
         endpoint_url: the S3 storage endpoint
         access_key: the S3 access key
         secret_key: the S3 secret key
-        bucket_name: the bucket name
         proxy_url: the proxy url
     """
 
-    def __init__(
-        self, region: str, endpoint_url: str, access_key: str, secret_key: str, bucket_name: str, proxy_url: str
-    ) -> None:
+    def __init__(self, region: str, endpoint_url: str, access_key: str, secret_key: str, proxy_url: str) -> None:
         _session = boto3.Session(access_key, secret_key, region_name=region)
         self._s3 = _session.client("s3", endpoint_url=endpoint_url)
         # Ensure S3 is connected
         try:
-            self._s3.head_bucket(Bucket=bucket_name)
+            self._s3.head_bucket(Bucket="admin")
```

Comment:
I would suggest we make this a hardcoded config variable (just so we make sure that it's uniform across the entire app & controlled by a single source of truth)

Here for instance there is a discrepancy with the initialization script I think :)

Reply:
hum indeed. Do we really want to create a bucket ? it should not even be used